Makefile

```diff
@@ -108,6 +108,10 @@ all: k0s k0s.exe
 go.sum: go.mod
 	$(GO) mod tidy
 
+codegen_targets = \
+	static/gen_manifests.go \
+	pkg/assets/zz_generated_offsets_$(shell go env GOOS).go
```

Comment:
Is there any way you can delegate to `$(GO)` to get this `go env` information? All of the main build (excluding tests + ci-stuffs) rely on the containerized go version.

Reply:
Coincidentally, that line broke the windows build in subtle ways, and the required thing to do here was to use `$(TARGET_OS)` instead of $(shell go env GOOS)`.

I see the point in not mixing the Go tooling of the host with the Docker tooling. This also happens for the ci-deps binaries, the unit tests and some other things.... Maybe it's worth to open an issue so we can discuss a bit.